### ISSUE TITLE
Swipe to unlock while using AOD

### ISSUE THREAD

**Issue by AhRickkyyyyyy** [straight-tamago/misakaX#314]
Devices after iPhone 14 Pro has the accessibility to swipe to unlock their phone while enabling AOD. I wish to add this accessibility to other phones which enables AOD too as this is quite useful. Now in the settings there is an option to let us swipe to unlock but does not seem to be working.

**Comment by TedKim317** [comment]
Hi, what model of phone are you using

**Comment by AhRickkyyyyyy** [comment]
> Hi, what model of phone are you using

I’m using 13 Pro Max

**Comment by TedKim317** [comment]
Try enabling the tap to wake feature for iPhone SE in misaka x

**Comment by AhRickkyyyyyy** [comment]
> Try enabling the tap to wake feature for iPhone SE in misaka x

Doesn't seem to work for me

**Comment by TedKim317** [comment]
Did it resolve the issue

**Comment by AhRickkyyyyyy** [comment]
> Did it resolve the issue

Nope, still can't swipe to unlock straight from AOD

**Comment by alanlu01** [comment]
I have same problem.
iPhone 13 mini

**Comment by TedKim317** [comment]
Yeah, just remembered iPhone 14 pro which supports the pad has a dedicated hardware in the chip for aod

**Comment by Newhellow-Realring** [comment]
Mine 15plus, also doesn’t work on swipe to unlock and tap to wake neither
